Tools/Scripts/hooks/pre-commit

```diff
@@ -4,9 +4,7 @@ import os
 import subprocess
 import sys
 
-ENCODING_KWARGS = dict()
-if sys.version_info >= (3, 6):
-    ENCODING_KWARGS = dict(encoding='utf-8')
+ENCODING_KWARGS = dict(encoding='utf-8')
```

Comment:
We should probably just feed this straight into the `subprocess` call now, although this is probably better as a follow-up fix.